StrataTest/DDM/Integration/Java/TestGen.lean

```diff
@@ -0,0 +1,302 @@
+/-
```

Comment:
Test 11 seems problematic since it uses javac. Is that even available in CI? I don't see it being installed in the CI script. How it is possible that this test passes in CI?

I think it would be better to have generated `.java` files checked into source and have a test compare those against what's currently generated.